handsontable/src/plugins/collapsibleColumns/collapsibleColumns.js

```diff
@@ -169,6 +169,11 @@ export class CollapsibleColumns extends BasePlugin {
         });
 
       } else if (Array.isArray(collapsibleColumns)) {
+
+        this.headerStateManager.mapState(() => {
```

Comment:
I'm thinking about covering this change with the test. Can you do that?

Reply:
It looks like there already is a test case that takes care of that -> https://github.com/handsontable/handsontable/blob/292374ff029a514ef73d7e0e78fc6100d9066a9b/handsontable/src/plugins/collapsibleColumns/__tests__/collapsibleColumns.spec.js#L368

It fails without this change. Before this PR it was passing by a (sort of) mistake - the Nested Headers plugin was clearing something, that should be cleaned by the Collapsible Columns plugin as well. When not all the plugins get updated on `updateSettings`, this problem emerged by itself.